routes/index.js

```diff
@@ -4,26 +4,37 @@ const projects = require('../database/db.js')
 
 
 /* GET home page. */
-router.get('/', function(req, res, next) {
-  res.render('index', { title: 'Express' });
-});
-
-router.get('/hai', (req, res) =>{
-  res.json({ title: 'HAI' });
+router.get('/', function(request, response, next) {
+  response.redirect('/getAllProjects');
 });
 
 
 router.get('/getAllProjects', (request, response) => {
-  projects.getAll()
-    .then(fromDB => response.render(index, {allProjects: fromDB}))
-    .catch(error => response.json(error))
+  projects.getAllProjects()
+    .then(fromDB => {
```

Comment:
Better convention to name this parameter after the data you expect to be passed to the callback (e.g., call it `allProjects`) instead of `fromDB`.